### ISSUE TITLE
ko

### ISSUE THREAD

**Issue by lipamanka** [lipu-linku/pali-nimi#22]
#### sona pu

NOUN clay, clinging form, dough, semi-solid, paste, powder

#### sona Linku pi toki Inli

clay, clinging form, dough, semi-solid, paste, powder

#### sona Linku pi toki pona

ko li kiwen ale ala li telo ale ala; sijelo ko li kama ante tan pana wawa

#### sona ku

goo⁵, semisolid⁵, paste⁴, powder⁴, clay³, sand², dust², substance², cream², dough², mud²

#### sona sin

**Comment by lipamanka** [comment]
my proposal: 
`semisolid, powder, paste, e.g. sand, soil, clay; squishy, moldable, fluffy, spongy`
it's a bit wordy but i think it works for ko. i have the nouns first and then adjectives after a semicolon. am i missing anything?

**Comment by RetSamys** [comment]
I'm in favour of "soft" over "fluffly, spongy" personally

**Comment by lipamanka** [comment]
i considered that. if something is hard wood but i put a thin layer of faux-fur on it, is it ko? i don't think so. soft is a surface thing, and the surface of something can be soft without it being ko. soi think it's not a good idea. but if you have a problem with the other two words, can you explain why? maybe it would be easier to comment more on them

**Comment by lipamanka** [comment]
oh almost forgot: convo topics from @AcipenserSturio 
> "semi-solid" and "clinging form" are notoriously confusing for learners and translators alike
> add goo at least
> maybe adjectives: malleable? soft?

let me rework my definition to incorporate these ideas

**Comment by RetSamys** [comment]
> but if you have a problem with the other two words, can you explain why? maybe it would be easier to comment more on them

👍 I guess my only issue with spongy is that it seems a bit specific, but not by much, so it's not that much of an issue. Fluffly is an interesting word, it doesn't fit into my brain yet - I don't know if it's just a me thing or if it's generally an odd word for people who didn't grow up with English. To get a better idea, I did what helped me in the past: I tried to look it up in English taso dictionaries as well as English<>German ones... All I'm getting is Urban Dictionary 😆 

Anyway, spongy and fluffly seemed to have "soft" in common, so replacing both with that seemed elegant, but if soft is more of a surface thing, then never mind.

**Comment by Th3Scribble** [comment]
> semisolid, powder, paste, e.g. sand, soil, clay; squishy, moldable, fluffy, spongy

the semicolon shouldn't be used here
i don't get the use of e.g. here, are all the translations after it examples of paste?

**Comment by RetSamys** [comment]
> but if you have a problem with the other two words, can you explain why? maybe it would be easier to comment more on them

👍 I guess my only issue with spongy is that it seems a bit specific, but not by much, so it's not that much of an issue. [Originally had a comment about "fluffly", but it was actually "fluffy".] I would have thought that fluffy is even closer to a surface thing than soft, but I could easily have the wrong impression about the word.

Anyway, spongy and fluffy seemed to have "soft" in common, so replacing both with that seemed elegant, but if soft is more of a surface thing, then never mind.

> > semisolid, powder, paste, e.g. sand, soil, clay; squishy, moldable, fluffy, spongy
>
> the semicolon shouldn't be used here
> i don't get the use of e.g. here, are all the translations after it examples of paste?

Maybe this would be better:

> semisolid, e.g. paste, powder, sand, soil...

**Comment by gregdan3** [comment]
I really like this ^ version with semisolid leading a bunch of examples

Personally don't have any issue with "spongy" or "soft" or "fluffy", but if it's a choice between them, I think we should pick at least 2 of the 3 but not soft+fluffy together (leans too hard toward uses of suwi)

How about this:

> semisolid, e.g. paste, powder, sand, soil, squishy, spongy, moldable, soft

**Comment by AcipenserSturio** [comment]
I don't like "soil" because it's clearly a ma - which is supported by ku data. Notably the same doesn't extend to sand or clay.

I still want "goo". Also, to a lesser extent, "malleable".

I understand *why* we are using "semisolid", its a really good shortcut, but it's very unfortunate the word isn't used pretty much at all outside tp and doesn't have any obvious translations.

We might want a semicolon between nouns and adjectives here. Just in terms of vibes

**Comment by lipamanka** [comment]
the semicolon that I had there was separating nouns and adjectives, which I explained in my first message, and then a bunch of people said that it didn't need to be there without explaining why so I'm inclined to kind of ignore that and keep it in for the time being lol. the semicolon means that the things that follow aren't examples anymore, and adjectives can't really be examples of a noun. 

I like soil because even if ma can mean soil, it shows overlap and it helps provide a through-line with clay and sand. Together those three demonstrate the idea that ko remains a ko no matter how much moisture there is in the semisolid, as long as it remains a semisolid. 

So to make a handful of adjustments to jan Kekan San's `semisolid, e.g. paste, powder, goo, sand, soil, clay; squishy, moldable, spongy, fluffy`
- added goo (for @AcipenserSturio) 
- added clay (to demonstrate the through-line with sand and soil) 
- added back the semicolon
- replaced soft with fluffy (for reasons I elaborated on above -  soft is moreso a surface thing, fluffy is moreso a consistency thing) 

I'd be fine with removing spongy, but I prefer it in there.

**Comment by Th3Scribble** [comment]
the semicolon is for different senses of the word and i don't think it's applicable here, since the only difference is that the adjective translations are "X" and the noun definitions are "thing that is X". it's describing the same qualities, it just so happens that some of the english equivalents are nouns and some are adjectives. i also don't think the e.g. applies since only paste, goo, and clay are semisolid (unless semisolid means something different and specific, but in that case i don't think we should use semisolid as the prime umbrella term anyway)

**Comment by janPensa** [comment]
> soft is moreso a surface thing, fluffy is moreso a consistency thing

Personally, I'd say that it's the inverse. Fluffy objects make me think mostly of the surface. Soft can refer to texture, but usually also consistency. You can have a solid object that "feels" soft but can't be compressed, but usually soft objects are compressible.

Looks like Oxford Dictionary and Wiktionary definitions go both ways.

Wiktionary gives
> **soft**
> 1. Easily giving way under pressure.
> 2. (of cloth or similar material) Smooth and flexible; not rough, rugged, or harsh.

and

> **fluffy**
> 1. Covered with fluff
> 2. Light; soft; airy.

In the Oxford Dictionary, the definition for "soft" has both "easy to mould, cut, compress, or fold" and after a semicolon "not hard or firm to the touch" (although "firm" implies consistency as well). And "fluffy" has
"of fluff", "like fluff" on one hand, and "covered with fluff" on the other.

IMO "easily giving way under pressure." and "easy to mould, cut, compress, or fold" seem appropriate for more different kinds of "ko" than "light; soft; airy" and what comes down to "made of fluff, similar to fluff".

Think of soft sand, soft clay, soft paste, soft-serve ice cream, etc. I might call soft serves fluffy, but not the others.

**Comment by lipamanka** [comment]
honestly this post is making me reconsider having fluffy or soft at all. they're too... idiomatic in english. Some ideas: 
1. We can use hyponyms (words that are "subsets" of other words) such as "furry" or "downy" (the examples listed in wiktionary). 
2. We can use "fluff" as a noun. this one is a substance and works pretty well. 

re: semicolon. 
> it's describing the same qualities, it just so happens that some of the english equivalents are nouns and some are adjectives.

I think that this is significant because when people are using a dictionary they're often inserting words and translating the sentence word by word into english in order to understand it better, so adding a semicolon helps smooth out the process for beginners. I know I did that when I was learning, and I still do that when I learn languages. If we're going to be listing nouns and adjectives I'd like to put a semicolon between them for a learner's sake.

**Comment by wasokeli** [comment]
i feel like "moldable, fluffy, spongy" are all implied by "squishy", which i think is a good word

**Comment by lipamanka** [comment]
what do we think about "fluff" as a noun?

**Comment by wasokeli** [comment]
feels a bit off to me. when i hear fluff, i think linja. even if they're all tangled

**Comment by lipamanka** [comment]
how might cotton fit into that? or wool? i guess i've never thought about how those might be a type of linja instead of a type of ko, but i still think because they can be a type of ko including them here is cool because it shows that there's some semantic overlap. if people feel *very strongly* that it doesn't belong in ko at all though i'm fine with keeping it out.

**Comment by gregdan3** [comment]
if you put powders or goops into a balloon, you get a squishy balloon

if you put fluffs (my catch all for cottons, wools, types of teddy bear stuffing) into a len, you get a squishy len

fluff(y) itself feels misleading (which i just put on the discord lol), but squishy captures it just fine i think!

**Comment by lipamanka** [comment]
do we have an agreeable def to rocket?

**Comment by wasokeli** [comment]
> semi-solid, e.g. paste, powder, goo, sand, soil, clay; squishy

**Comment by janPensa** [comment]
Having just "squishy" seems weird. I'd add at least one or two from the previously discussed "malleable, moldable, spongy, soft, fluffy".

Soft and either moldable or malleable would have my personal preference.

**Comment by gregdan3** [comment]
i don't think squishy either works alone (there are more words to describe different vibes of ko) or should be excluded (for the reasoning given in my last comment; it does capture an important property of ko)

i am against soft for similar reasons to being against fluff(y); i also think "squishy" captures it sufficiently

i am split on including "spongy", but right now i think it is similar enough to squishy

 but i do feel like "moldable" or "malleable" are both meaningfully different from squishy and one of them should be included. they both apply well to the types of ko that maintain the form you put them into, like clays; squishy doesn't make much of a claim about form before/after interaction, although arguably it implies the ko would return to its pre-existing form.

and of moldable/malleable, i like moldable more; that's based on my personal vibe: to me, malleable feels more related to metals, and moldable feels more related to clays.

> semi-solid, e.g. paste, powder, goo, sand, soil, clay; squishy, moldable

so, this is my proposal!
